QueueUsingStack.java

```diff
@@ -0,0 +1,49 @@
+// Time Complexity : O(1) for every operation, amortized for push
```

Comment:
Amortized would be when O(n) operations occurs once and then we get O(1) operation cost for the next **n** calls.
In this case for 10 back to back push operations, we only have O(1) for one call and the rest 9 calls are O(n).
Please review the Time Complexity again for push.